brainstormforce/astra
# AST-2567 - Feat: Date extended options for meta fields for single post type

### Description
- Previously we used to ask users to add custom code snipett as per follow doc
- https://wpastra.com/docs/show-last-updated-not-published-date/
- Now with this dynamic setup we are coming up new 2 options
- - Date type
- - Date format

### Screenshots
- https://share.bsf.io/RBuBXJe9
- Implementation - https://share.bsf.io/qGuZDqmA

### Types of changes
- New feature

### How has this been tested?
- Check master to this branch conversion
- Check for single post, as in verify the options provided
- Check for CPT single layouts as well 

### Checklist:
